packages/polar/src/internal/cli/project-creation.ts

```diff
@@ -47,15 +49,21 @@ function copySampleProject (location: string): void {
   });
 }
 
-function printSuggestedCommands (): void {
+function printSuggestedCommands (projectName: string): void {
+  const currDir = process.cwd();
+  const projectPath = path.join(currDir, projectName);
+  console.log(`Success! Created project at ${chalk.greenBright(projectPath)}.`);
+  // console.log(`Inside that directory, you can run several commands:`);
```

Comment:
can we remove this line or uncomment it ?

Reply:
This I've left if we later we want to add description of all commands later. But if not, I can remove it.

Reply:
ok, Please add a TODO: ... here